### PR TITLE
Fix double reference.

### PR DESCRIPTION
**Pull request by jashapiro**
Found a very minor error in the intro... filing this as a sorta-hotfix to the merge PR, will also merge to staging. We'll see if this works.

### CODE REVIEW

**Comment by jashapiro** [comment]
The main fix is line 66.

**Comment by jaclyn-taroni** [comment]
Oops, I merged the other one because I didn't know about this.

**Review by cansavvy — APPROVED** [review]
🚀 This "warm" fix seems fine.

**Comment by jashapiro** [comment]
Let me refile as a proper hotfix.

File: 01-getting-started/getting-started.Rmd [hunk 1]
```diff
@@ -23,24 +23,24 @@ output:
...
+This tutorial has follow-along examples for use with refine.bio downloads.
 
 ## About how this tutorial book is structured 
 
 This tutorial contains follow-along analysis examples for refine.bio gene expression data.
-The analysis examples are organized by technology: ["microarray"](https://alexslemonade.github.io/refinebio-examples/02-microarray/intro-to-microarray.nb.html) or ["RNA-seq"](https://alexslemonade.github.io/refinebio-examples/03-rna-seq/intro-to-rna-seq.nb.html), in addition to an ["Advanced Topics"](https://alexslemonade.github.io/refinebio-examples/04-advanced-topics/intro-to-advanced-topics.nb.html) section. 
-Each analysis is self-contained and provides information with how to obtain the dataset used in the example from refine.bio. 
+The analysis examples are organized by technology: ["microarray"](https://alexslemonade.github.io/refinebio-examples/02-microarray/intro-to-microarray.nb.html) or ["RNA-seq"](https://alexslemonade.github.io/refinebio-examples/03-rna-seq/intro-to-rna-seq.nb.html), in addition to an ["Advanced Topics"](https://alexslemonade.github.io/refinebio-examples/04-advanced-topics/intro-to-advanced-topics.nb.html) section.
```
Comment (jaclyn-taroni) [inline 01-getting-started/getting-started.Rmd:35]: While we're at it....

```suggestion
The analysis examples are organized by technology: ["microarray"](https://alexslemonade.github.io/refinebio-examples/02-microarray/intro-to-microarray.html) or ["RNA-seq"](https://alexslemonade.github.io/refinebio-examples/03-rna-seq/intro-to-rna-seq.html), in addition to an ["Advanced Topics"](https://alexslemonade.github.io/refinebio-examples/04-advanced-topics/intro-to-advanced-topics.html) section.
```

Comment (jashapiro) [reply 01-getting-started/getting-started.Rmd:35]: implemented in #444

Comment (jaclyn-taroni) [reply 01-getting-started/getting-started.Rmd:35]: Thank you!

**Pull request closed without merging**